rhessys/cycle/surface_daily_F.c

```diff
@@ -189,6 +207,21 @@ void		surface_daily_F(
 	hv = (2.5023e6 - 2430.54 * zone[0].metv.tday) / 1000; /* changed to match hv used in penman monteith */
 	water_density = 1000;	/* density of water in kg/m^3 */
 	
+	double daylength = zone->metv.dayl;
+	double nightlength = SECONDS_PER_DAY - daylength;
+	double day_proportion = daylength / SECONDS_PER_DAY;
+
+	double surface_heat_flux_day = day_proportion * patch->surface_heat_flux;
+	double surface_heat_flux_night = patch->surface_heat_flux - surface_heat_flux_day;
+
+	double rain_duration_day = zone->daytime_rain_duration * day_proportion;
```

Comment:
This really should be done in zone_daily_F - that way if you are running hourly and know the
actual hours when it rained - you could use those to compute the proportions of the day vs night 
that rain occurred in - it is also more consistent with RHESSys structure to do these met type calcs there
(and so they could be used by other routines)